gce_vars/machines

```diff
@@ -1,4 +1,4 @@
 ---
 # Common Compute Engine instance variables
 machine_type: n1-standard-1
-image: debian-7
+image: https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/debian-9-stretch-v20181210 
```

Comment:
Can you use an image family here so it stays current longer?

Reply:
Good call!